src/Tests/Microsoft.Diagnostics.Monitoring.Tool.UnitTests/EndpointInfoSourceTests.cs

```diff
@@ -214,7 +214,7 @@ public static IEnumerable<object[]> GetTfmsSupportingPortListener()
             yield return new object[] { TargetFrameworkMoniker.Net60 };
         }
 
-        private ServerEndpointInfoSource CreateServerSource(out string transportName, ServerEndpointInfoCallback callback = null)
+        internal ServerEndpointInfoSource CreateServerSource(out string transportName, ServerEndpointInfoCallback callback = null)
```

Comment:
If these are useful as utility methods, refactor them to a separate class that can be used by multiple test classes. Test classes should not be calling to each other. Then the utility class can have public methods that are clearly meant to be consumed by other code.